client/scss/components/_gif_video.scss

```diff
@@ -37,15 +25,20 @@
   }
 }
 
-.gif-video__button-text {
-  @include visually-hidden();
+.gif-video__play {
+  .gif-video__play-pause.is-playing & {
```

Comment:
Why not a modifier? I still don't see when to, and when not to, use BEM?

Reply:
Another question here about what's the point of nesting this? Another thing BEM solves.

Reply:
As a rule of thumb, I like using `is-blah` classes for state changes brought about via JS – it lends itself to being easier to abstract across JS modules. But given the JS for this module is not in the least bit abstract, I'll bend to your wishes!